--- kchmviewer-1.3/kio-msits/kchmviewer.desktop~	2005-06-24 10:30:27.000000000 +0200
+++ kchmviewer-1.3/kio-msits/kchmviewer.desktop	2006-01-26 11:01:17.000000000 +0100
@@ -4,8 +4,9 @@
 Type=Application
 Exec=kchmviewer %U
 Icon=kchmviewer.png
-Comment=
-Terminal=0
+Terminal=false
 Name=KchmViewer
 Comment=A viewer of CHM (MS-Help) files
+Comment[pl]=Przeglądarka plików CHM (Pomoc MS)
 MimeType=application/chm
+Categories=Qt;Office;Viewer;
